src/neuroglancer/segmentation_user_layer.css

```diff
@@ -99,6 +99,18 @@
   user-select: text;
 }
 
+.neuroglancer-segment-list-entry .neuroglancer-color-widget {
+  border: none;
+  border-color: transparent;
+  -webkit-appearance: none;
+  background-color: transparent;
+  padding: 0;
+  margin: 0;
+  margin-left: 3px;
+  height: 19px;
+  width: 20px
+}
+
```

Comment:
example of the styling
<img width="149" alt="Screen Shot 2022-02-22 at 3 55 43 PM" src="https://user-images.githubusercontent.com/188155/155217853-73f76ff7-f6f3-40ab-89a8-3ddf2e0f39cd.png">

Reply:
Since the color is already displayed on the id itself, and space is tight in the segment list, I think it would be better to just use a modifier+click, e.g. alt+click, to change the color without having any visible control.

Reply:
after talking with our group, while that seems very clean, we think the discoverability of the feature would be too limited with that approach

Reply:
In that case perhaps we can use a build-time option to enable the visible widget, but still allow a modifier+click to change the color in any case?